docs/root/start/quick-start/run-envoy.rst

```diff
@@ -82,6 +82,9 @@ Run Envoy with the demo configuration
 
 The ``-c`` or ``--config-path`` flag tells Envoy the path to its initial configuration.
 
+Envoy will parse the config file according to the file extension, please see the
+:ref:`command line options <operations_cli#cmdoption-c>` for further information.
```

Comment:
ci is back - it doesnt like this link

Reply:
i think change `` :ref:`command line options <operations_cli#cmdoption-c>` `` to  `` :option:`config path command line option <-c>` ``

not 100% sure - but i think that should work

Reply:
I will try it.